app/controllers/api_controller/tags.rb

```diff
@@ -7,7 +7,8 @@ module Tags
     #
 
     def tags_query_resource(object)
-      object ? object.tags.where(Tag.arel_table[:name].matches "#{TAG_NAMESPACE}%") : {}
+      # object ? object.tags.where(Tag.arel_table[:name].matches "#{TAG_NAMESPACE}%") : {}
```

Comment:
I wonder if something else is not working with /api/chargebacks w.r.t. tags.
if it's not /managed, try searching by tag,

https://github.com/ManageIQ/manageiq_docs/blob/master/api/examples/queries.adoc#query-services-tagged-for-the-finance-department

```
GET /api/chargebacks?by_tag=/assigned_to/..
```

We may need to support both namespaces where TAG_NAMESPACE is checked.